add_me_to_your_PATH/julia_pod

```diff
@@ -142,7 +151,7 @@ kubectl apply -f $DRIVER_YAML -n "$KUBERNETES_NAMESPACE"
 
 log "Waiting for job to have active pod..."
 
-until kubectl get "job/$RUNID" -n "$KUBERNETES_NAMESPACE" -o 'jsonpath={..status.active}' | grep '1'; do
+until kubectl get "job/driver-$RUNID" -n "$KUBERNETES_NAMESPACE" -o 'jsonpath={..status.active}' | grep '1'; do
```

Comment:
is this a bug on master? Needed this to get it to work but I don't think my changes introduce the issue

Reply:
Could be that you need to `rm driver.yaml.template` in the project dir you're using, to let `julia_pod` use the latest version of the default template?

Reply:
yep that was it, thanks!